.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Setup rust toolchain
         run: rustup show
       - uses: Swatinem/rust-cache@v1.3.0
-      - run: cargo build --bins --tests
+      - run: cargo build --features expensive_tests --bins --tests
       - run: cargo test --workspace
```

Comment:
I think you need to add it to the test target aswell. Otherwise the test is built but not running. 

```
      - run: cargo test --workspace --features expensive_tests
```